content/backend/graphql-python/6-error-handling.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Error Handling
+description: Handling Errors on GraphQL
+---
+
+All applications fail, and GraphQL it's no different. Some clients may ask for information that's not available or execute a forbidden action. In this chapter, you'll understand how GraphQL and Graphene address these issues.
+
+### Schema Errors
+Being a typed system, GraphQL can predetermine if a query is valid. All the fields from queries and mutations have a strong type, so requesting and inputting wrong data will generate an error.
```

Comment:
"Being a typed system, GraphQL can predetermine if a query is valid. " --> I would say this is a bit inaccurate. GraphQL _is_ a typed _language_ (not really a system) - or you could say GraphQL is a language with a strong type system.